sdk/python/packages/flet/src/flet/canvas/__init__.py

```diff
@@ -1,12 +1 @@
-from flet.core.canvas.arc import Arc
-from flet.core.canvas.canvas import Canvas, CanvasResizeEvent
-from flet.core.canvas.circle import Circle
-from flet.core.canvas.color import Color
-from flet.core.canvas.fill import Fill
-from flet.core.canvas.line import Line
-from flet.core.canvas.oval import Oval
-from flet.core.canvas.path import Path
-from flet.core.canvas.points import PointMode, Points
-from flet.core.canvas.rect import Rect
-from flet.core.canvas.shadow import Shadow
-from flet.core.canvas.text import Text
+
```

Comment:
why removing this?

Right now to work with Canvas you have to do:

```py
import flet.canvas as cv
```

See here example: https://github.com/flet-dev/examples/blob/main/python/controls/canvas/canvas-face.py

Reply:
Oh, my bad, this is unintentional - thanks for the catch. 
My IDE has this "optimize imports" functionality which removes all unused imports in the files being committed.

Will rollback.